src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final int COLOR_ENUM_LENGTH = Color.values().length;
+    private static final Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        Color[] colors = Color.values();
+        return colors[random.nextInt(COLOR_ENUM_LENGTH)].toString();
```

Comment:
```suggestion
        return colors[random.nextInt(COLOR_ENUM_LENGTH)].name();
```
see in Common mistakes
you should use `name()` method to get String value of an enum
because `toString()`z can be overridden and will work not as you expect it